docs/source/model_design/designing_vivarium_model/interactive_sim/index.rst

```diff
@@ -56,13 +56,124 @@ This makes some tasks more difficult to complete interactively.
+It will be a .yaml file. 
+
+If you can't locate the proper model specs file or are unsure - ask the engineers for help! 
+
+Once you have that, you can follow this code to create a simulation 
+
+::
+
+  path = Path('<file_path_for_model_spec_file>')
+  
+  sim = InteractiveContext(Path(path), setup=False)
+
+This creates an object called 'sim' that is the simulation. 
+
+**Step 3: Updating the Simulation Parameters**
+
+Often, you will want to use slightly different parameters than are the standard in the model 
+spec. For example, you might want to change the population size. To do this, you can use 
+'configuration.update'. 
+
+Below is an example of changing the population size from 100,000 in the model spec file 
+to be 50,000 for the interactive sim. You can use this to help change anything in the 
+'configuration' section of the model spec. 
+
+These are the lines we will be editing as written in the model spec file: 
+
+:: 
+
+  configuration: 
+    population: 
+      population_size: 100_000
+      age_start: 5 
+      age_end: 125 
+
+Now, we will update the 'population_size' variable to be 50,000 in our notebook: 
+
+::
+
+  sim.configuration.update({
+                          'population':
+                              {'population_size': 50_000,
+                              },})
+
+Instead of using 'configuration.update' you can just directly change the values in the 
+model spec file. However, since this file exists in the engineering repo, changing it on 
+your local machine can cause errors and confusion. Also, whenever you pull from main, 
+your updates would be removed. Therefore, we do NOT recommend this approach. 
+
+**Step 4: Loading Data from the Simulation** 
+
+Now that you have the parameters set-up, you're ready to start getting data from the simulation. 
+The first step is to run 'sim.setup'. Running this command will take some time and possibly generate 
+some pink warning text. Don't worry! Just wait for the cell to finish running. 
+
+Once it's done, you will have a simulated population. You can use 'get_population' to create a dataset 
+with your population. Some simulant data automatically gets recorded for your sim. To find a list of these, 
+list the columns in your dataset. Other simulant data does not automatically get added, but can also 
+be saved. To find a list of additional sim data available, use 'list_values'. If you find 
+something from the list that you want included in your data, just add it using 'get_value'. 
+
+The below will show using all of these in practice: 
+
+:: 
+
+  sim.setup() # Sets up the simulation 
+  pop0 = sim.get_population() # Generates a dataset with some simulant data included 
+  pop0.columns # Lists the columns in your simulant dataset 
+  sim.list_values() # Lists the additional columns you can add to the dataset 
+  sim.get_value('<variable_from_list_values>')(pop0.index) #Pulls data for all simulants 
```

Comment:
Should this show how to join it into the pop0 dataframe? That's what I would typically do.